tests/plugins/balloonpanel/_helpers/tools.js

```diff
@@ -119,6 +119,23 @@ var balloonTestsTools = {
 				break;
 		}
 		return pos;
+	},
+
+	removeDottsFromUrl: function( expectedPath ) {
+		var parts = expectedPath.split( '/' ),
```

Comment:
The implementation would be much simpler if you'd use `CKEDITOR.tools.array.filter`.

Reply:
I'm not sure if implementation would be simpler. Please notice that I need to remove also folder names in the path not only dots.
So when there is path like: `/one/two/three/../../four`, I need to receive `/one/four`. This is why I look on array from right to left and preserve information how many addtional folders have to be removed.
To achieve similar result I would need to revers array. Then have filter function in clouser, which will preserve information how many additoianl folders need to be removed. And then revers array again. I don't feel like using `CKEDITOR.tools.array.filter` will simplify thi for loop.